content/zh-cn/docs/concepts/architecture/garbage-collection.md

```diff
@@ -20,7 +20,7 @@ allows the clean up of resources like the following:
 垃圾收集允许系统清理如下资源：
 
 <!--
-* [Failed pods](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection)
+* [Terminated pods](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection)
 * [Completed Jobs](/docs/concepts/workloads/controllers/ttlafterfinished/)
```

Comment:
There is another PR already working on it.
- #35787
(Made by the en PR author, and 8 hours before this PR open)

Reply:
> Hold until conflicted PR merged and rebase this one.

Thanks. I have rebased it, PTAL.